packages/editor/src/translation/form-editor/en.json

```diff
@@ -0,0 +1,249 @@
+{
+  "align": {
+    "bottom": "Bottom",
+    "center": "Center",
+    "top": "Top"
+  },
+  "button": {
+    "description": "A button to fire actions",
+    "name": "Button",
+    "type": {
+      "danger": "Danger",
+      "primary": "Primary",
+      "secondary": "Secondary"
+    }
+  },
+  "category": {
+    "actions": "Actions",
+    "behaviour": "Behaviour",
+    "columns": "Columns",
+    "content": "Content",
+    "dynamicOptions": "Dynamic Options",
+    "elements": "Elements",
+    "formatting": "Formatting",
+    "general": "General",
+    "hidden": "Hidden",
+    "input": "Input",
+    "layout": "Layout",
+    "options": "Options",
+    "paginator": "Paginator",
+    "parameters": "Parameters",
+    "selection": "Selection",
+    "staticOptions": "Static Options",
+    "structures": "Structures",
+    "styling": "Styling",
+    "text": "Text"
+  },
+  "checkbox": {
+    "description": "A selectable boolean checkbox",
+    "name": "Checkbox"
+  },
+  "combobox": {
+    "addDropdown": "Add Dropdown-Button to Combobox",
+    "description": "A autocomplete combobox with label",
+    "name": "Combobox"
+  },
+  "composite": {
+    "description": "A button for fire actions",
+    "name": "Composite"
+  },
+  "dataTable": {
+    "columnFromValue": "Create columns from value",
+    "description": "A datatable component",
+    "name": "DataTable",
+    "valueEmpty": "Value of DataTable is empty. Define value within Properties of DataTable"
+  },
+  "dataTableColumn": {
+    "description": "A Column for the DataTable",
+    "name": "DataTableColumn"
+  },
+  "datePicker": {
+    "description": "A datepicker with label for date or datetime",
+    "label": "Date Picker",
+    "name": "DatePicker"
+  },
+  "dialog": {
+    "description": "A flexable layout",
+    "name": "Dialog"
+  },
+  "fieldset": {
+    "description": "A group of inputs",
+    "name": "Fieldset"
+  },
+  "hint": {
+    "dragFirstItem": "Drag first element inside the canvas or",
+    "dragItemInto": "Drag first element inside the {{type}}"
+  },
+  "hotkey": {
+    "createFormData": "Create from data ({{hotkey}})",
+    "deviceMode": "Change device mode ({{hotkey}})",
+    "focusInscription": "Focus Inscription ({{hotkey}})",
+    "focusMain": "Focus Main ({{hotkey}})",
+    "focusToolbar": "Focus Toolbar ({{hotkey}})",
+    "openDataClass": "Open Data Class({{hotkey}})",
+    "openHelp": "Open Help ({{hotkey}})",
+    "openProcess": "Open Process ({{hotkey}})",
+    "redo": "Redo ({{hotkey}})",
+    "undo": "Undo ({{hotkey}})",
+    "viewMode": "View Mode ({{hotkey}})"
+  },
+  "input": {
+    "description": "A input with a label",
+    "name": "Input"
+  },
+  "label": {
```

Comment:
- check for common labels
- move component specific labels to component block
- think about global component block with all components in it